src/remoteplayer.h

```diff
@@ -32,6 +32,16 @@ enum RemotePlayerChatResult
 	RPLAYER_CHATRESULT_KICK,
 };
 
+struct CloudSettings
```

Comment:
Sorry to change my suggestion but, i think 'cloud params' is better as is shorter and more consistent with other code, 'parameters' is an annoyingly long word to type.